src/carma-1-tenth/plot_localization.py

```diff
@@ -62,8 +62,8 @@ def plot_localization(bag_dir, show_plots=True):
                 particle_std_deviations[particles_count] = np.diagonal(np.sqrt(np.cov(particles.T, aweights=weights)))
                 particle_times[particles_count] = t_
                 particles_count += 1
-            elif topic == cmd_vel_topic:
-                velocities[velocities_count] = msg.linear.x
+            elif topic == vel_topic:
+                velocities[velocities_count] = msg.twist.twist.linear.x
```

Comment:
There shouldn't be any difference between the twist and ackermann msg velocity but since ackermann is more directly feeding into the system I'd use that unless you think otherwise

Reply:
Switched to ackermann

Reply:
Ended up reverting back to `/cmd_vel` since there is no `/ackermann_cmd` in the turtlebot sim